### PR TITLE
feat(climate): Add Zehnder manufacturer synonym

### PR DESCRIPTION
**Pull request by mattheop**
Hi,
I just saw your PR on the ZHA library, which is used by the integration on Home Assistant. I just bought an Acova heater.

I saw in your MR that you put as manufacturers `“ZEHNDER GROUP VAUX ANDIGNY      ”.` I assume that this is not a mistake and that it's your actual heater manufacturer. 

Personally, mine does not contain the ending spaces. Maybe it has a more recent version, and ZEHNDER fix this typo ? So I propose this PR.

I've just tested it and everything works.

Thanks to your contribution that allow me to control my heater :)

### CODE REVIEW

**Comment by gdoffe** [comment]
Hi @mattheop,

I'm glad it's useful to someone.  
I developed this code somewhat empirically; I'm more familiar with embedded software. ^^'  

Thank you for your PR; it seems entirely relevant. Zehnder has most likely removed trailing spaces from strings in the newer versions.

**Comment by mattheop** [comment]
Hi @gdoffe,

I just pushed two commits with formatting and test fixes to validate the CI. Let me know if you have any feedback :)

**Review by gdoffe — APPROVED** [review]

**Comment by gdoffe** [comment]
Seems good to me, I will squash tonight commits to keep consistency, ok for you ?

**Comment by mattheop** [comment]
ok 👍

File: zha/application/platforms/climate/__init__.py [hunk 1]
```diff
@@ -589,6 +589,67 @@ class ZenWithinThermostat(Thermostat):
     """Zen Within Thermostat implementation."""
 
 
+@MULTI_MATCH(
+    cluster_handler_names=CLUSTER_HANDLER_THERMOSTAT,
+    manufacturers={"ZEHNDER GROUP VAUX ANDIGNY      "},
```
Comment (mattheop) [inline zha/application/platforms/climate/__init__.py]: maybe you have overwrite  #66e5d513b761f80ce3094806231f1678cd3a7a91 commit ?

**Comment by gdoffe** [comment]
I squashed commits and added you as co-author:
https://github.com/gdoffe/zha/commits/add_acova_heaters/

Ok to close here ?

**Comment by mattheop** [comment]
No problem for co-author and squash !
I remove my previous comment, I was on a wrong branch.

You can close, thank you for your contribution :)

**Comment by gdoffe** [comment]
Thanks to you, I hope it will be merged...

**Pull request closed without merging**